drivers/storage/portworx/util/csi_generator.go

```diff
@@ -190,6 +190,9 @@ func (g *CSIGenerator) GetCSIConfiguration() *CSIConfiguration {
 		cv.IncludeCsiDriverInfo = true
 	}
 
+	cv.IncludeAttacher = true
```

Comment:
Let's make this conditional and not set to true all the time.

Reply:
Not required for now.